projects/ad7768-evb/src/ad7768_evb.c

```diff
@@ -256,15 +255,6 @@ int main(void)
 	if (ret != 0)
 		return -1;
 
-	axi_gpio_init.type = GPIO_PL;
```

Comment:
I wouldn't remove this code yet. Rather check inline for the definition of `GPIO_STATUS_DEVICE_ID` and only if available perfrom these status checks.